Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Subsystems/WorldSubsystem.h"
+#include "PSWorldSubsystem.generated.h"
+
+/**
+ * Implements the world subsystem to access different components in the module 
+ */
+UCLASS(BlueprintType, Blueprintable, Config = "ProgressionSystem", DefaultConfig)
+class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
+{
+	GENERATED_BODY()
+
+public:
+	/** Returns this Subsystem, is checked and wil crash if can't be obtained.*/
+	static UPSWorldSubsystem& Get();
+	static UPSWorldSubsystem& Get(const UObject& WorldContextObject);
+
+	/** Returns the data asset that contains all the assets of Progression System game feature.
+	 * @see UPSWorldSubsystem::PSDataAssetInternal. */
+	UFUNCTION(BlueprintCallable, BlueprintPure, Category = "C++")
+	const class UPSDataAsset* GetPSDataAsset() const;
+	
+	/** Returns a progression System component reference */
+	UFUNCTION(BlueprintPure, Category = "C++")
+	FORCEINLINE class UPSHUDComponent* GetProgressionSystemComponent() const { return ProgressionSystemComponentInternal;	}
+
+	UFUNCTION(BlueprintCallable, Category = "C++")
+	void SetProgressionSystemComponent(UPSHUDComponent* MyProgressionSystemComponent);
```

Comment:
Any UFUNCTION, UPROPERTY, UCLASS, USTRUCT, UENUM have to have comment above, Please comment your function

Reply:
![image](https://github.com/h4rdmol/ProgressionSystem/assets/5227233/63ed8260-d4a9-497b-a55d-fa15356028a0)
can't reproduce

Reply:
found another missing places, added respective comment